python/hail/dataset.py

```diff
@@ -2000,6 +2000,37 @@ def impute_sex(self, maf_threshold=0.0, include_par=False, female_threshold=0.2,
         """Impute sex of samples by calculating inbreeding coefficient on the
         X chromosome.
 
+        **Examples**
+
+        >>> imputed_sex_vds = vds.impute_sex()
```

Comment:
Can you use the following example instead:

Remove samples where imputed sex does not equal reported sex:
```
>>> imputed_sex_vds = (vds.impute_sex()
...       .annotate_samples_expr('sa.sexcheck = sa.pheno.isFemale == sa.imputesex.isFemale')
...       .filter_samples_expr('sa.sexcheck || isMissing(sa.sexcheck)'))